highlighters/main/test-data/back-quoted-argument.zsh

```diff
@@ -27,8 +27,9 @@
 # vim: ft=zsh sw=2 ts=2 et
 # -------------------------------------------------------------------------------------------------
 
-BUFFER='echo `echo 42`'
+BUFFER='echo `echo \`42\`` `echo 6 times 9'
```

Comment:
Suggest adding a command that the «42» is in command position, so if someone ever tries to support nested quotes and sees this test fail, it'll be clear to them that it's a false positive.